help_to_heat/frontdoor/models.py

```diff
@@ -15,7 +15,13 @@ class Answer(utils.UUIDPrimaryKeyBase, utils.TimeStampedModel):
     session_id = models.UUIDField(editable=False)
 
     class Meta:
-        indexes = [models.Index(fields=["session_id"])]
+        indexes = [
+            models.Index(
+                name="idx_answer_session_id",
+                fields=["session_id", "page_name", "-created_at"],
```

Comment:
(I presume yes, unless it's a copy+paste error :D)

Reply:
Yeah on purpoise. 🐬